CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
+- Adjust payment terms on cost center
+
```

Comment:
We need to move these lines to under the `##[Unreleased]` section (before `## [1.11.2] - 2024-06-12`

Reply:
Hey @enzomerca, changes made!!